# generic issue to report
string issue_report

# flag indicating if issue includes relevant control info
bool has_control_info

# string containing any control info to report
string control_info

# flag indicating if issue includes operator suggestion
bool has_suggestion

# string containing any suggestions for operator
string suggestion
